tests/mock_data/calicoctl/mesh/static-routes/step2/input.yaml

```diff
@@ -0,0 +1,5 @@
+kind: BGPConfiguration
+apiVersion: projectcalico.org/v3
+metadata:
+  name: default
+spec:
```

Comment:
is this valid yaml? I'd expect this would need to be `spec: {}`

Reply:
I wondered that, but it seems to work this way too, and better IMO not to mix in JSON syntax.